src/main/scala/io/qbeast/spark/delta/CubeInfo.scala

```diff
@@ -20,9 +20,12 @@ import io.qbeast.core.model.Weight
 /**
  * Cube Information
  *
- * @param cube      Id of the cube
- * @param maxWeight Maximum maxWeight of the cube
- * @param size      Number of elements of the cube
+ * @param cube
+ *   Id of the cube
+ * @param maxWeight
+ *   Maximum maxWeight of the cube
+ * @param size
+ *   Number of elements of the cube
  */
 
 case class CubeInfo(cube: String, maxWeight: Weight, size: Long)
```

Comment:
I don't think this class is used anywhere

Reply:
Totally right. Deleting file 👍